Sources/HTTP/Response/ResponseRepresentable.swift

```diff
@@ -28,3 +30,14 @@ extension Swift.String: ResponseRepresentable {
         )
     }
 }
+
+///Allows Foundation Data to be returned through closures
+extension Foundation.Data: ResponseRepresentable {
+    public func makeResponse() -> Response {
+        return Response(
+            status: .ok,
+            headers: [:],
```

Comment:
Even if you don't explicitly set an empty array, there is still no content type being set.

The problem is I don't know what content type would be appropriate for a generic byte array.

Maybe this?

https://stackoverflow.com/questions/6783921/which-mime-type-to-use-for-a-binary-file-thats-specific-to-my-program